reference/api/index.rst

```diff
@@ -0,0 +1,180 @@
+Web API
+=======
+
+.. toctree::
+
+    rest
+    jobs
+
+How it's structured
+-------------------
+
+There is actually two APIs. A REST API with basic CRUD operations, and a jobs API which can launch heavy processes asynchronously.
+The current version of the API is v1. All URI have to request explicitly this version.
+
+.. code-block:: bash
+
+    GET https://demo.akeneo.com/api/rest/v1
+    GET https://demo.akeneo.com/api/jobs/v1
+
+Request these URIs to get all the endpoint categories that the API supports.
+
+Scope of the API
+----------------
+
+Currently, only Product, Category, Family, Attribute and Attribute Option entities are covered by the API.
+
+.. warning::
+    As these APIs are mainly designed to be used by connectors, Enterprise Edition permissions are not managed.
+
+Security
+--------
+
+We use `OAuth2 <https://oauth.net/2/>`_ to authenticate users in the API and
+`Symfony ACLs <http://symfony.com/doc/2.7/security/acl.html>`_ to handle authorizations.
+
+Authentication
+~~~~~~~~~~~~~~
+
+Creating an OAuth client
+************************
+
+The first operation you will need to authorize third-party applications to use the API is to create a `client id`.
```

Comment:
_The first operation to do is to authorize third-party applications to use the API. For that, you need to create a `client id`._